h/search/query.py

```diff
@@ -22,21 +22,19 @@ def wildcard_uri_is_valid(wildcard_uri):
     """
     Return True if uri contains wildcards in appropriate places, return False otherwise.
 
-    *'s are not permitted in the scheme or netloc. _'s are not permitted in the scheme.
+    *'s and _'s are not permitted in the scheme or netloc.
```

Comment:
`netloc` is a somewhat python-centric term (i.e. web devs might not recognize it as a concept immediately). Perhaps help define this somewhere in the documentation for this endpoint by defining a URI as being compose of `scheme://netloc/path;parameters?query#fragment` ? (At least, I presume you're using python `urlparse`/`urllib.parse` parsing and this is, thus, true).